lib/fixed-points.nix

```diff
@@ -41,6 +41,13 @@ rec {
   # think of it as an infix operator `g extends f` that mimics the syntax from
   # Java. It may seem counter-intuitive to have the "base class" as the second
   # argument, but it's nice this way if several uses of `extends` are cascaded.
+  #
```

Comment:
Needs a bit of introduction like "Let's see how `extends` evaluates the example."

Reply:
`g` itself should also be introduced as an example, which the pre-existing text fails to do.

Reply:
g is defined in the text above the context